### DISCUSSION TITLE
Maintain translation in another repo to avoid bloating the git history

### DISCUSSION THREAD

**Discussion by tuukkamustonen** [fastapi/fastapi#6572] [Questions] [answered]
### First Check

- [X] I added a very descriptive title to this issue.
- [X] I used the GitHub search to find a similar issue and didn't find it.
- [X] I searched the FastAPI documentation, with the integrated search.
- [X] I already searched in Google "How to X in FastAPI" and didn't find any information.
- [X] I already read and followed all the tutorial in the docs and didn't find an answer.
- [X] I already checked if it is not related to FastAPI but to [Pydantic](https://github.com/samuelcolvin/pydantic).
- [X] I already checked if it is not related to FastAPI but to [Swagger UI](https://github.com/swagger-api/swagger-ui).
- [X] I already checked if it is not related to FastAPI but to [ReDoc](https://github.com/Redocly/redoc).

### Commit to Help

- [x] I commit to help with one of those options 👆

### Example Code

```python
n/a
```

### Description

Rather than hosting the documentation translation in this repo, could they live somewhere else? It's a bit annoying skipping over the translation commits when you're investigating what _development_ has been going on.

Lots of the commits are translations and if [those are going to increase](https://github.com/tiangolo/fastapi/issues?q=is%3Aissue+is%3Aopen+translation) it's going to be unbearable at some point.

It's, of course, nice that documentation lives alongside code, but as translations reflect the English docs anyway, could there be an official fork which gets auto-updated on each commit, and where the translations get applied. This would keep the main repository free of bloat.

### Wanted Solution

Host the translations in another repo.

### Wanted Code

```python
n/a
```

### Operating System

Linux, Windows, macOS, Other

### FastAPI Version

unrelated

### Python Version

unrelated

**Comment by tiangolo** [comment, marked as answer]
Thanks for the interest! But I'll pass on this. Sorry, it would mean just even more work for me. And I like that contributions to documentation and translations are indeed contributions to FastAPI, the people that do it end up having commits in the main repo.

I'm also not familiar with your workflow with Git, I don't do that myself. But maybe you can orient yourself in the releases page: https://fastapi.tiangolo.com/release-notes/

Given that I'll close this, but thanks for the interest.

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
